metrics/grafana/dashboard.json

```diff
@@ -834,5 +873,5 @@
   "timezone": "",
   "title": "snarkOS node",
   "uid": "PAzNcaCGz",
-  "version": 53
+  "version": 55
 }
```

Comment:
nit: missing newline

Reply:
this is a straight copy from the `grafana` settings window, so on the off-chance that it dislikes a trailing newline, perhaps it's better to let it be :smiley: